src/window.c

```diff
@@ -65,7 +65,12 @@
 #include <sys/iosupport.h>
 #endif
 
+#ifdef CONFIG_NDS
+// Should be sufficient with a disabled editor.
+#define NUM_SAVSCR 3
+#else
 #define NUM_SAVSCR 6
+#endif
```

Comment:
Could this be generalized to `#ifndef CONFIG_EDITOR`? Not sure how many levels the help system and/or renderer switching can consume worst case but a more general ifdef might be better. (Alternatively, `CONFIG_NO_LAYER_RENDERING` already cut this array to a third its original size, so it might be safest to just leave this at 6.)

Reply:
Actually, this is fine for now. It can be reverted later if it causes issues down the road.